_data/glossary/customresourcedefinition.yaml

```diff
@@ -0,0 +1,10 @@
+name: Custom Resource Definition
+aka: Third Party Resource
+tags:
+full-link: docs/tasks/access-kubernetes-api/migrate-third-party-resource/
+- fundamental
+- operation
+short-description: >
+  A custom resource definition is the representation of a customization of a particular Kubernetes installation.
```

Comment:
short description should be a fragment. When we automatically add it to a docs topic, the first part is automatically added. So should start "The representation of ..."
But ... this is also pretty thin as a short description. You can customize a K8s cluster lots o' ways -- what makes a CRD different? (Feel free to ping on Slack if you have questions)